partiql-ast/src/experimental/ast.rs

```diff
@@ -23,6 +23,13 @@ pub struct Item {
     // and how it contributes to backward compatibility.
 }
 
+impl fmt::Display for Item {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
```

Comment:
nit: signature from the [docs](https://doc.rust-lang.org/std/fmt/trait.Display.html) is slightly different. I don't think it should matter, though.

```
fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result
```

Reply:
Yep, in this case we don't use the _lifetime_ but I'll change it in a new commit.